index 4243d02c..af7ebe6d 100644
--- a/firedrake/tsfc_interface.py
+++ b/firedrake/tsfc_interface.py
@@ -144,1 +144,1 @@ SplitKernel = collections.namedtuple("SplitKernel", ["indices",
                                                      "kinfo"])


-def compile_form(form, name, parameters=None, inverse=False, split=True, interface=None, coffee=False, diagonal=diagonal):
+def compile_form(form, name, parameters=None, inverse=False, split=True, interface=None, coffee=True, diagonal=diagonal):
     """Compile a form using TSFC.
 
     :arg form: the :class:`~ufl.classes.Form` to compile.
